src/main/java/teammates/common/datatransfer/FeedbackTextResponseDetails.java

```diff
@@ -15,28 +17,28 @@ public FeedbackTextResponseDetails() {
     
     public FeedbackTextResponseDetails(String answer) {
         super(FeedbackQuestionType.TEXT);
-        this.answer = answer;
+        this.answer = answer == null ? null : Sanitizer.sanitizeForRichText(answer);
```

Comment:
@damithc , as I see, a null-check was added to Sanitizer.sanitizeForRichText(). It returns null if input parameter is null. So, this ternary operator has become useless